src/Stryker.Core/Stryker.Core/Reporters/HtmlReporter/HtmlReporter.cs

```diff
@@ -14,13 +15,13 @@ public class HtmlReporter : IReporter
     {
         private readonly StrykerOptions _options;
         private readonly IFileSystem _fileSystem;
-        private readonly IChalk _chalk;
+        private readonly TextWriter _output;
```

Comment:
I think the name _output is a bit misleading. IMO `_consoleWriter` would be a better fit. The same goes for the other classes that use this.

Reply:
I thought to keep it a little bit more abstract, as you're not always writing to a console (f.e. in the unit tests).
But I will change this